### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kilrau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,7 @@ Download the latest binaries from the [releases](https://github.com/BoltzExchang
 
 ### Docker
 
-`boltz-client` is also available as [docker image](https://hub.docker.com/r/boltz/boltz-client/tags).
-
-Example usage where your lnd macaroon and certificate are placed inside `~/.lnd`
-
-To start the build process, run:
```
Comment (jackstar12) [inline README.md:27]: This isnt building the image, its using our published version.

Comment (kilrau) [reply README.md:27]: yes that's why I removed it?

Comment (jackstar12) [reply README.md:27]: thought it was the other way because i cant remember writing it. seems like it got in there during merging

Comment (kilrau) [reply README.md:27]: prbly it was me, don't worry about it

Comment (kilrau) [reply README.md:27]: merge?

**Pull request merged**